Dockerfile.builder.arm64

```diff
@@ -0,0 +1,91 @@
+FROM ubuntu:18.04 as bazel-builder
+LABEL maintainer="maintainer@cilium.io"
+
+RUN mkdir -p /go/src/github.com/cilium/cilium/envoy
+WORKDIR /go/src/github.com/cilium/cilium/envoy
+COPY . ./
+
+#
+# Install Bazel
+#
+
+RUN apt-get update && apt-get install -y unzip zip curl
+
+
+RUN export BAZEL_VERSION=`cat BAZEL_VERSION` && echo $BAZEL_VERSION && \
+     curl -sfL https://github.com/Tick-Tocker/bazel-arm64/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-aarch64 \
```

Comment:
I have the same concern here as for the other PR:

https://github.com/cilium/proxy/pull/18#discussion_r408480935

Whatever we do to resolve this, we should make it consistent across the repository.

Reply:
@joestringer Thank you. 
I had replied this in https://github.com/cilium/proxy/pull/18#discussion_r408480935.
In summary, it's a concise and workable solution here. In case it's not workable in the future, we can still deal with it by somthing like "build from source" or others. 
Actually I had tested a "build from source" solution for this Dockerfile and it's also ok to use, but just want to get a concise and quick way here for Bazel arm64.

Reply:
@TrevorTaoARM me and Joe will meet this week to discuss this. We will provide feedback.

Reply:
@TrevorTaoARM Can you use a bazel package for arm64, if available, from ubuntu? Or if we only need the binary would the package from open suse work? cc @mrostecki 

Also, can you create a `Dockerfile.builder.multi-arch` that could accept an ARCH as an argument? Later on we would deprecate `Dockerfile.builder` and replace it with the `Dockerfile.builder.multi-arch`

Reply:
I can create a deb package for Debian and Ubuntu on openSUSE Build Service, based on the RPM spec we have. Give me one day, I will ping you once it's ready and post instructions how to install it.

I'm also about to submit an update to Bazel 3.0 today. :)

Reply:
Thanks for the offer @mrostecki , I think that this should resolve the remaining concerns:
* Fast turn-around to unblock other work
* Low long-term maintenance by reusing work openSUSE folks are already doing
* Easier to validate the integrity of the build chain

Reply:
@aanm @mrostecki @joestringer Thank you.
Yes, if the Bazel binary/pkg is available (soon) from ubuntu or openSUSE, I would use it. 
I would also try to create a Dockerfile.builder.multi-arch with an ARCH argument. It would take all platforms together so it may need some changes on current work.